Shared/Theme/Themeable.swift

```diff
@@ -21,11 +22,19 @@ extension Themeable {
         themeObserver = notificationCenter.addObserver(name: .ThemeDidChange,
                                                        queue: mainQueue) { [weak self] _ in
             self?.applyTheme()
-            self?.updateThemeApplicableSubviews()
         }
     }
 
-    public func updateThemeApplicableSubviews() {
+    public func listenForThemeChange(_ subview: UIView) {
```

Comment:
Old method for listenForThemeChange() still exists. I am thinking of removing it. Any objections cc @OrlaM

Reply:
removed `listenForThemeChange()`